### PR TITLE
Add README, LICENSE, and gulpfile to bower ignore list

### LINKED ISSUE #68: Add README and gulpfile.js to bower ignore?
Currently the [/bower.json `ignore` array](https://github.com/mozilla/tabzilla/blob/8b5e990fa4e9309c2e23fdb44fc125e0e5bd225f/bower.json#L10-L19) looks like this:

```js
  "ignore": [
    "docs-assets",
    "less",
    "node_modules",
    ".gitignore",
    "bower.json",
    "Gruntfile.js",
    "index.html",
    "package.json"
  ],
```

Not sure if we want to add the README.md (1.7KB) and the gulpfile.js (620B) to the `ignore` list. We can probably just change the existing Gruntfile.js with gulpfile.js since it looks like we introduced Gulp in #54 PR.

```
➜  tabzillaz  bower i mozilla-tabzilla
bower mozilla-tabzilla#*        cached git://github.com/mozilla/tabzilla.git#0.3.0
bower mozilla-tabzilla#*      validate 0.3.0 against git://github.com/mozilla/tabzilla.git#*
bower mozilla-tabzilla#~0.3.0  install mozilla-tabzilla#0.3.0

mozilla-tabzilla#0.3.0 bower_components/mozilla-tabzilla

➜  tabzillaz  ls -lash bower_components/mozilla-tabzilla
total 32
0 drwxr-xr-x  8 pdehaan  staff   272B Aug  4 13:36 .
0 drwxr-xr-x  3 pdehaan  staff   102B Aug  4 13:36 ..
8 -rw-r--r--  1 pdehaan  staff   862B Aug  4 13:36 .bower.json
8 -rw-r--r--  1 pdehaan  staff   1.7K Jul 15 01:27 README.md
8 -rw-r--r--  1 pdehaan  staff   574B Jul 15 01:27 bower.json
0 drwxr-xr-x  3 pdehaan  staff   102B Jul 15 01:27 css
8 -rw-r--r--  1 pdehaan  staff   620B Jul 15 01:27 gulpfile.js
0 drwxr-xr-x  3 pdehaan  staff   102B Jul 29 17:43 media
```

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #68

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -13,9 +13,11 @@
     "node_modules",
     ".gitignore",
     "bower.json",
-    "Gruntfile.js",
+    "gulpfile.js",
     "index.html",
-    "package.json"
+    "package.json",
+    "LICENSE",
```
Comment (pdehaan) [inline bower.json]: I added the LICENSE file in #67.

File: bower.json [hunk 2]
```diff
@@ -13,9 +13,11 @@
     "node_modules",
     ".gitignore",
     "bower.json",
-    "Gruntfile.js",
+    "gulpfile.js",
```
Comment (schalkneethling) [inline bower.json]: woops, good catch!

**Comment by schalkneethling** [comment]
All good except, not sure about the README. I felt it would be pretty useful to have that available offline. Thoughts?

**Comment by pdehaan** [comment]
@schalkneethling I figured the only people using this repo would be Mozillians who either know about the project or know how to find the GitHub repo if they care about the repo. But that said, it's only 1.7KB and I'm extremely indifferent. I'm happy to add the README back in and squash some commits in the morning if you prefer.

**Comment by schalkneethling** [comment]
sounds good to me @pdehaan Thanks!

**Comment by pdehaan** [comment]
@schalkneethling removed README.md from the ignore list.

**Comment by schalkneethling** [comment]
r+ :fried_shrimp:

**Comment by pdehaan** [comment]
@schalkneethling Thanks for all your help with this!
Any chance we can get a 0.4.0 tagged and pushed to bower+npm? :heart:

**Comment by schalkneethling** [comment]
@pdehaan sure thing, will look into that today.

**Pull request merged**